.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,15 +6,15 @@ to the issue. -->
 
 # Checklist
 
-- [ ] Commits in meaningful sequence and with useful messages
-- [ ] Tests added or updated when needed
-- [ ] `CHANGELOG.md` files updated for packages with externally visible changes<br>
-      **_New section is never added with the code changes._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
+- [ ] Commits in meaningful sequence and with useful messages.
+- [ ] Tests added or updated when needed.
+- [ ] `CHANGELOG.md` files updated for packages with externally visible changes.
+      **NOTE: _New section is never added with the code changes._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
```

Comment:
I don't have any opinions on the new line issue,  but it bothers me a little that some lines end with a full stop and some not :laughing:   
I have a slight preference for no full stops, but if we add them, then maybe let's add them for each item ?